cli/commands/new.js

```diff
@@ -48,9 +49,9 @@ async function createPackage(packageName, target = "node") {
 
     process.chdir(packageName);
 
-    // FIXME This is messing clio.toml. Will remain commented until dependency management is fixed
-    // await packageConfig.fetchDependencies();
-    // info("Added Clio dependencies");
+    // FIXME This is messing <CONFIGFILE_NAME>. Will remain commented until dependency management is fixed
```

Comment:
What's up with this fixme? The line is not commented anymore. Can we remove it?

Reply:
yes I was waiting for feedback, I'll remove!